winmake.ps1

```diff
@@ -48,7 +48,8 @@ function Local-Unit {
     Build-Ginkgo
     $skippackages="hack,internal\domain\infra\abi,internal\domain\infra\tunnel,libpod\lock\shm,pkg\api\handlers\libpod,pkg\api\handlers\utils,pkg\bindings,"
     $skippackages+="pkg\domain\infra\abi,pkg\emulation,pkg\machine\apple,pkg\machine\applehv,pkg\machine\e2e,pkg\machine\libkrun,"
-    $skippackages+="pkg\machine\provider,pkg\machine\proxyenv,pkg\machine\qemu,pkg\specgen\generate,pkg\systemd,test\e2e,test\utils,cmd\rootlessport"
+    $skippackages+="pkg\machine\provider,pkg\machine\proxyenv,pkg\machine\qemu,pkg\specgen\generate,pkg\systemd,test\e2e,test\utils,cmd\rootlessport,"
+    $skippackages+="pkg\pidhandle"
```

Comment:
if the unit tests don't run on windows then they should have !windows build tags instead of skipping them here like this

Reply:
Although your test file is already called linux_test.go so it should already be only called on linux so I don't quite follow why this would be needed

Reply:
No, I don't know how to do that. For some reason, it fails to compile the package because "build constraints exclude all Go files". The only way I found to avoid adding the package in the skip list is to add a `pidhandle_windows.go` that only has the package def:

```golang
//go:build windows

package pidhandle
```

Reply:
What about using `//go:build !remote`? The `remote` is a remote client. Am I right?

Reply:
`//go:build !remote` or `//go:build !windows` have the same effect (on windows) and don't fix the problem.